Apps/Src/ReadCarCAN.c

```diff
@@ -171,7 +107,7 @@ void Task_ReadCarCAN(void *p_arg)
         0,
         CAN_WATCH_TMR_DLY_TMR_TS,
         OS_OPT_TMR_PERIODIC,
```

Comment:
@diyarajon Me, Maddie and Roie had a discussion about this briefly. We actually do want this to be periodic, since we need the saturation buffer to be decremented every single time we miss a charge enable message (i.e. whenever we reset the watchdog timer). Make sure you are also decrementing the saturation buffer in the callback for this.